cookbooks/prescribed_velocity/corner_flow.prm

```diff
@@ -5,6 +5,11 @@ set End time    = 0
 # Load the signal library.
 set Additional shared libraries = ./libprescribed_velocity.so
 
+subsection Discretization
```

Comment:
Instead of modifying an existing test, can you just clone it? This way, we test both the existing code path as well as the new one of this patch.